src/v/v8_engine/tests/scripts/sum.js

```diff
@@ -1,4 +1,4 @@
 function sum(obj) {
     let array = new Int32Array(obj);
     array[2] = array[0] + array[1];
-}
\ No newline at end of file
+}
```

Comment:
This change to add a newline can be in the previous commit

Reply:
done